webpack/components/AnsibleRoles/AnsibleRolesPage.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import TableIndexPage from 'foremanReact/components/PF4/TableIndexPage/TableIndexPage';
+import { translate as __ } from 'foremanReact/common/I18n';
+import { getDocsURL } from 'foremanReact/common/helpers';
+import { AnsibleRolesImportButton } from './components/AnsibleRolesImportButton';
+import { getActions } from '../AnsibleRoles/AnsibleRolesPageHelpers';
+
+export const AnsibleRolesPage = () => (
+  <TableIndexPage
+    header={__('Ansible Roles')}
+    controller="ansible_roles"
+    apiUrl="/ansible/ui_ansible_roles"
+    apiOptions={{ key: 'WEBHOOKS_API_REQUEST_KEY' }}
```

Comment:
Please change to an ansible key instead of webhooks

Reply:
Whoops, that shouldn't be there...